convert/github/convert_test.go

```diff
@@ -26,7 +26,7 @@ import (
 )
 
 func TestConvert(t *testing.T) {
-	tests, err := filepath.Glob("testdata/*/*.yaml")
+	tests, err := filepath.Glob("testdata//*.yaml")
```

Comment:
change to `testdata/*/*/*.yaml`

Reply:
This looks like an accidental change in the github logic, I removed it from this PR.

`testdata/*/*.yaml` should be correct since there is only one level of directories under `github/testdata`.